auth.ts

```diff
@@ -0,0 +1,46 @@
+import NextAuth from "next-auth"
+import Google from "next-auth/providers/google"
+import Credentials from "next-auth/providers/credentials"
+import type { Provider } from "next-auth/providers"
+
+ 
+const providers: Provider[] = [
+  Credentials({
+    credentials: { password: { label: "Password", type: "password" } },
+    authorize(c) {
+      if (c.password !== "password") return null
+      return {
+        id: "test",
+        name: "Test User",
+        email: "test@example.com",
+      }
```

Comment:
Is this intended to stay in the code base?
I'm not sure what it does but, it seems like it may be test specific.
However, in my manual testing everything seems to work just fine 🤷🏽